### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by pbnj**
Per your recommendations:

- default search results limit to 5
- use https & satisfy linter warnings by adding comments above exported variables (closes #6 )
- bump semver patch to `0.0.3` (`0.0.2` being the introduction of `--open`)
- Update readme

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -13,11 +12,73 @@ Via `go-get` or the binary [releases](https://github.com/tj/go-search/releases).
...
+      Options:
+        -n, --count n    number of results [default: 5]
+        -t, --top        top-level packages only
+        -o, --open       open godoc.org search results in default browser
+        -h, --help       output help information
+        -v, --version    output version
+    ```
+- Examples:
+    ```sh
+    $ go-search go-debug
```
Comment (tj) [inline Readme.md]: Should the arg here be just "debug"?

**Review by tj — COMMENTED** [review]
Cool LGTM other than I thinkkkk that arg should remove the go- prefix

File: Readme.md (hunk 1 above)
Comment (petermbenjamin) [inline Readme.md]: oh `go-debug` is the name of the package I'm searching for. Sorry, that's probably a confusing example.

Comment (petermbenjamin) [inline Readme.md]: I will change `go-debug` in the example to another package name to avoid confusion.

Comment (tj) [inline Readme.md]: Ahhh I just wasn't expecting Godocs to return the others with the prefix, interesting that it does. "UUID" is a good one hahaha I search for that all the time

**Comment by petermbenjamin** [comment]
Done. Replaced `go-debug` with `UUID` in readme examples.

**Comment by tj** [comment]
thanks :D

**Pull request merged**